x/staking/keeper/delay_updates.go

```diff
@@ -0,0 +1,29 @@
+package keeper
+
```

Comment:
Main changes are the refactoring of validator update delay checks into this file.